### PR TITLE
Fix various small issues in Pike CLI

### PR DESCRIPTION
**Pull request by dplumb94**

### CODE REVIEW

File: cli/src/main.rs [hunk 1]
```diff
@@ -320,28 +320,18 @@ fn run() -> Result<(), CliError> {
...
                                     .required(true)
-                                    .help("Public Key and unique identifier for agents"),
+                                    .help("Public key and unique identifier for agents"),
                             )
                             .after_help(AFTER_HELP_WITHOUT_KEY),
                     )
                     .subcommand(
                         SubCommand::with_name("list")
                             .about("List all agents for a given service")
-                            .arg(
```
Comment (KevinAnsleyJohnson) [inline cli/src/main.rs:335]: This is one that I tried removing before and Shawn said it should stay - basically allows the --service-id to come before or after the arguments... Although this is not standard to the rest of the cli

**Review by KevinAnsleyJohnson — COMMENTED** [review]
Awesome!!!

Just a quick note on one area.

File: cli/src/main.rs (hunk 1 above)
Comment (dplumb94) [reply cli/src/main.rs:335]: Fixed by making the the service-id and url args global one level up from here.

File: cli/src/main.rs [hunk 2]
```diff
@@ -185,6 +185,7 @@ fn run() -> Result<(), CliError> {
                     Arg::with_name("service_id")
                         .long("service-id")
                         .takes_value(true)
+                        .global(true)
```
Comment (KevinAnsleyJohnson) [inline cli/src/main.rs:188]: very cool!

File: cli/src/main.rs [hunk 3]
```diff
@@ -528,7 +528,7 @@ fn run() -> Result<(), CliError> {
                                 .takes_value(true)
                                 .multiple(true)
                                 .use_delimiter(true)
-                                .help("List of organizations allowed use of the role"),
+                                .help("List of org IDs of organizations allowed use of the role"),
```
Comment (peterschwarz) [inline cli/src/main.rs:537]: Is "org" redundant?

Comment (vaporos) [reply cli/src/main.rs:537]: I agree with this - everything else LGTM

**Review by peterschwarz — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**